svc-managers/managers/managers.go

```diff
@@ -501,7 +513,7 @@ func fillResponse(ctx context.Context, body []byte, managerData map[string]inter
 	var respData map[string]interface{}
 	err := json.Unmarshal([]byte(data), &respData)
 	if err != nil {
-		l.LogWithFields(ctx).Error(err.Error())
+		l.LogWithFields(ctx).Error("unable to unmarshal : " + err.Error())
```

Comment:
err.Error() also give same error , no need to additional message

Reply:
done